src/Generator/Passes/FixCommentsPass.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+using CppSharp.AST;
+
+namespace CppSharp.Passes
+{
+    /// <summary>
+    /// Pass that makes Xml Documentation Comments almost Xml Documentation Comments again.
+    /// </summary>
+    /// <remarks>
+    /// But has a bunch of para cruft that doesn't seem to be removable.
+    /// </remarks>
+    public class FixCommentsPass : TranslationUnitPass
+    {
+        private readonly Regex m_rx = new Regex(@"///(?<text>.*)", RegexOptions.ExplicitCapture | RegexOptions.Compiled);
+
+        public override bool VisitDeclaration(Declaration declaration)
+        {
+            if (AlreadyVisited(declaration))
+                return false;
+
+            //if (declaration.TranslationUnit == null || !declaration.TranslationUnit.FileName.StartsWith("Chakra"))
+            //    return false;
+
+            if (declaration.Comment != null)
+            {
+                var xDoc = GetOriginalDocumentationDocument(declaration.Comment.Text);
+                var xRoot = xDoc.Root;
+
+                declaration.Comment.Kind = CommentKind.BCPLSlash;
```

Comment:
I actually left everything as it was in this Pass' original writer's version. Just change something here. He used `RawCommentKind.BPCLSlash` from `CppSharp.Parser.AST` but seeing the type of left hand expression is not that. I changed it to `CommentKind.BPCLSlash` from `CppSharp.AST`. I hope it's okay.  
  
Also, we should mention somewhere that this code was taken from another repository...